modules/apps/object/object-service/service.xml

```diff
@@ -248,6 +248,7 @@
 		<column name="rootObjectEntryId" type="long" />
 		<column name="defaultLanguageId" type="String" />
 		<column name="treePath" type="String" />
+		<column name="expirationDate" type="Date" />
 		<column name="version" type="int" />
 		<column name="lastPublishDate" type="Date" />
 		<column name="status" type="int" />
```

Comment:
During the refinement session we decided to create these columns (`statusByUserId`, `statusByUserName`, `statusDate`). Did you forget to create them?
If we have v1 (approved) and v2 (approved), when a user expires v1, how can we know who expired this version?
You can test this on the Web Content screen to better understand their behavior.

Reply:
Just done !